pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTest.java

```diff
@@ -71,7 +77,9 @@ private List<Object[]> queryRunner(String sql) {
         for (ServerInstance serverInstance : queryPlan.getStageMetadataMap().get(stageId).getServerInstances()) {
           DistributedStagePlan distributedStagePlan =
               QueryDispatcher.constructDistributedStagePlan(queryPlan, stageId, serverInstance);
-          _servers.get(serverInstance).processQuery(distributedStagePlan, requestMetadataMap);
+          EXECUTOR_SERVICE.submit(() -> {
```

Comment:
just wondering, what's the motivation for this change?

Reply:
ah. this was me trying to simulate an actual submission of stage.
- previously they all run in the main test thread
- unit-test it doesn't include the query server, which suppose to run on a remote service

so this is to simulate a simultaneous submission and execution without the server / serde utils.